src/providers/front.ts

```diff
@@ -0,0 +1,74 @@
+// Website: https://front.com/product/live-chat
+// Documentation: https://help.front.com/
+
+import { State } from '../types'
+import waitForLoad from '../utils/waitForLoad'
+
+declare global {
+  interface Window {
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    FrontChat: any
```

Comment:
Is this a constant that Front sets?

Reply:
Yep.  Should I pull more detailed types from Front's sdk or are we happy with `any` here?

Reply:
Nope, this is ok. Was just making sure I understood 👍

Reply:
Went ahead and added them for completeness 😄